src/Core/Grid/Definition/Factory/RequestSqlGridDefinitionFactory.php

```diff
@@ -91,7 +107,7 @@ protected function getColumns()
                 ->setOptions([
                     'filter' => new ColumnFilterOption(SearchAndResetFormType::class, [
                         'attr' => [
-                            'data-url' => '',
+                            'data-url' => $this->resetSearchUrl,
```

Comment:
you need to rebase develop and use `data-redirect`

Reply:
done.